.travis.yml

```diff
@@ -16,5 +16,7 @@ matrix:
       env: TOXENV=flake8
     - python: 2.7
       env: TOXENV=ansible-lint
+    - python: 2.7
+      env: TOXENV=ansible2-lint
```

Comment:
An alternative approach to using a separate testenv in tox would be to set up another environment variable, e.g., `ANSIBLE_VERSION=">=2.0"`

And then have `ansible-lint` simply do something like this for requirements:

```
passenv=ANSIBLE_VERSION
deps=
    -rtest-requirements.txt
    ansible{env:ANSIBLE_VERSION:>=1.9.1,<2.0}
```